src/code/z_player_lib.c

```diff
@@ -778,35 +778,38 @@ s32 func_8008F2BC(Player* this, s32 itemAction) {
     return -1;
 }
 
-s32 func_8008F2F8(PlayState* play) {
+s32 Player_GetEnvHazard(PlayState* play) {
```

Comment:
we can possibly afford the full "EnvironmentHazard" here, and only use env where it needs shortening (like in enum prefixes)

Reply:
I see you put `Environmental` first before reverting, tbh it might be better as environmental. it makes it more of an adjective instead of just "environment hazard" which could help separate it form the actual environment system

Im fine with either, but just wanted to mention it